env/payment_mongodb.env

```diff
@@ -1,4 +1,4 @@
 MONGODB_DATABASE: payment_mongodb
```

Comment:
I think that username and password can be removed, as we dont use that anymore in the connection config

Reply:
Will it work without username and password using kubernetes?

Reply:
It should. These variables were only used in the app.config afaik. The current version of that string does not include an username or password. From the logs of the shards it seems that everyone can change data

Reply:
They can be removed. Everyone can change data yeah, so probably a security risk but no big deal ;)

Reply:
Could you remove them then ;)